apps/authentication/backends/oidc/backends.py

```diff
@@ -61,8 +61,7 @@ def get_or_create_user_from_claims(self, request, claims):
         logger.debug(log_prompt.format("user: {}|created: {}".format(user, created)))
         logger.debug(log_prompt.format("Send signal => openid create or update user"))
         openid_create_or_update_user.send(
-            sender=self.__class__, request=request, user=user,
-            created=created, attrs=user_attrs,
+            sender=self.__class__, user=user, created=created, attrs=user_attrs,
         )
         return user, created
 
```

Comment:
I have no knowledge cutoff after September 1st, 2021. Please give another question about which you would like help with coding or technical details of specific programming problems or solutions.